### ISSUE TITLE
Map multiple controller buttons to same action

### ISSUE THREAD

**Issue by franciscolourenco** [OpenEmu/OpenEmu#1037]
This "feature" already occurs if you map a physical button to an action and then map a different button to that same action, leaving the first button unmapped.
In this case both buttons activate the same action, even though only the second button is explicitly mapped. 

This is desirable since in most modern controllers there are more buttons than actions and it is practical to have the extra buttons mapped to the same actions to use in different situations or games.

However when OpenEmu is restarted, any button which was not visibly mapped in the preferencies, doesn't trigger any action.

This issue is for the implementation of persistence across application restart of this behaviour, which even though probably accidental, seems to be desirable.

**Comment by iparr** [comment]
I was driven mad by this issue, maybe this will help:
https://www.reddit.com/r/OpenEmu/comments/2bniuo/set_bindings_to_the_same_keyboard_key/cj7qbji

**Comment by mattlondon** [comment]
The "just works" simplification means Chrono Trigger doesn't work, at least not with a keyboard.

Early in the game, the player must press L Bumper, R Bumper, and A Button simultaneously. Keyboards can't do this. One solution is to bind all of the buttons to the same key temporarily. Sadly, this is impossible in OpenEmu.

Is Chrono Trigger played on a keyboard an edge case?

**Comment by iparr** [comment]
Did you read the post? I temporarily bound keys at the other end of each keyboard and was able to progress through Chrone Trigger.

**Comment by mattlondon** [comment]
Yes, I did read it, but apparently I hadn't picked the right key bindings because I just retried with Q, B, and \ and it's working! The adventure continues...

**Comment by mrvacbob** [comment]
Which keyboards work with which keys depends on the individual model, unfortunately.
This is why I have a stockpile of Apple Extended Keyboard IIs!

**Comment by ron-wolf** [comment]
The option to map keys to (timed?) button presses using JSON files or some other human-readable format might help with this.

**Comment by prophittcorey** [comment]
This same problem bit me in Chrono Trigger. I recommend at least putting a note in the readme or somewhere.

**Comment by acambera-nearsoft** [comment]
> Yes, I did read it, but apparently I hadn't picked the right key bindings because I just retried with Q, B, and \ and it's working! The adventure continues...

This worked for me. Thanks!. I was stuck trying to play Chrono Trigger

**Comment by SuperTurn** [comment]
Another example of this being useful is during the end part of Final Fantasy 8. You have to hold down 8 buttons (X, Square, Circle, Triangle, R1, R2, L1 and L2) at the same time when playing the organ in Ultimecia's Castle. I tried mapping all eight buttons to the letter A (per the original reply listed above which states that actions mapped to a key retain their key mapping when you map another action to that key), and this did not work. 
<img width="592" alt="Screen Shot 2020-09-06 at 2 26 35 PM" src="https://user-images.githubusercontent.com/6256019/92335578-f95ca300-f04c-11ea-8e8f-1b1af062d2a5.png">

**Comment by clobber** [comment]
@SuperTurn 

The issue you are experiencing has to do with key "[rollover](https://en.wikipedia.org/wiki/Rollover_(key))". Apple keyboards are **NOT** N-Key rollover, so depending on the combination of keys pressed on certain rows, of course they won't all register.

Instead, use a gamepad.
